docs/dev-env-first-time-contributors.md

```diff
@@ -696,6 +713,18 @@ christie@xenial:~
 $ sudo patch --directory /usr/lib/ruby/vendor_ruby/vagrant < vagrant-plugin.patch
 patching file bundler.rb
 ```
+#### Lost Connectivity or power during vagrant setup
+If you see the following when you run `vagrant ssh` on Ubuntu:
```

Comment:
Provisioning can fail on Windows/Cygwin and OS X as well as Ubuntu. I suggest describing the issue like you did earlier -- the missing of the python virtualenv prompt `(zulip-env)` after `vagrant ssh`.